src/hotspot/src/cpu/aarch64/vm/assembler_aarch64.hpp

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 1997, 2019, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2013, 2019, Red Hat Inc. All rights reserved.
+ * Copyright (c) 1997, 2021, Oracle and/or its affiliates.
```

Comment:
No need for a copyright update for backports. This one is dated 2015, so the existing 2019 is fine.

Reply:
I update copyright headers.